.bra.toml

```diff
@@ -1,19 +1,19 @@
 [run]
 init_cmds = [
-  ["go", "run", "-mod=vendor", "build.go", "-dev", "build-cli"],
```

Comment:
Does this change mean that we no longer use the vendor folder when building grafana in bra? And does that have any potential side effects?

Reply:
@xlson Good question, but the behaviour should be the same as before because Go 1.14 uses the `-mod=vendor` mode [automatically](https://golang.org/doc/go1.14#go-command) when the vendor directory is present.

Reply:
We agreed in the team to remove `-mod=vendor` after upgrading to 1.14.